_data/toc.yaml

```diff
@@ -3053,7 +3053,7 @@ manuals:
         - sectiontitle: Sign images
           section:
           - path: /ee/dtr/user/manage-images/sign-images/
-            title: Sign an image
+            title: Using Notary to sign an image
```

Comment:
The linked page is technically a guide to using the "docker trust" cli commands to sign.  Before these trust commands were added to the CLI, advanced signing operations were done with the Notary Client.  I think in some cases the notary client still needs to be used, so we might add an "advanced signing" link to https://docs.docker.com/notary/advanced_usage/